shell/src/main/java/tachyon/shell/command/RmCommand.java

```diff
@@ -65,4 +65,9 @@ void runCommand(TachyonURI path) throws IOException {
   public String getUsage() {
     return "rm <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Removes the specified file.";
```

Comment:
I guess that this comment should be regarding RmrCommand.java which is bellow. I'll place that.